axelor-stock/src/main/resources/domains/StockMove.xml

```diff
@@ -99,6 +99,24 @@
 	
 		public static final String ORIGIN_INVENTORY = "com.axelor.apps.stock.db.Inventory";
 	]]></extra-code>
+	
+	<track>
+      <field name="stockMoveSeq" />
+      <field name="typeSelect" />
+      <field name="partner" />
+      <field name="statusSelect" />
+      <field name="fromStockLocation" />
+      <field name="toStockLocation" />
+      <field name="estimatedDate" />
+      <field name="realDate" if="statusSelect == 3" />
+      <field name="originTypeSelect" if="originTypeSelect != null" />
+      <field name="exTaxTotal" />
+      <message if="true" on="CREATE">Move created</message>
+      <message if="statusSelect == 1" tag="important">Draft move</message>
+      <message if="statusSelect == 2" tag="info">Planned move</message>
+      <message if="statusSelect == 3" tag="success">Move realized</message>
+      <message if="statusSelect == 4" tag="success">Move canceled</message>
+    </track>
 
```

Comment:
Remove 'exTaxTotal', 'originTypeSelect', 'also' separate messages are there for 'statusSelect', so no need to put it as field. Add message for cancel status.